metricbeat/module/golang/expvar/_meta/data.json

```diff
@@ -0,0 +1,60 @@
+{
+    "@timestamp": "2017-02-10T07:50:49.746Z",
+    "beat": {
+        "hostname": "MacBook-Pro.local",
+        "name": "MacBook-Pro.local",
+        "version": "6.0.0-alpha1"
+    },
+    "golang": {
+        "expvar": {
+            "cmdline": "./metricbeat -httpprof=127.0.0.1:6060 -e -v",
+            "fetches": {
+                "golang-expvar": {
+                    "events": 1,
+                    "failures": 0,
+                    "success": 1
+                },
+                "golang-heap": {
+                    "events": 1,
+                    "failures": 0,
+                    "success": 1
+                }
+            },
+            "libbeat.config.module.running": 0,
```

Comment:
This kind of keys will not work with 2.x releases. But that is something we should fix in the beats itself to return proper json ( @urso I remember you were doing something in this direction?). 

For services that provide this kind of keys, we can either replace all `. ` with `_` or from my perspective nicer would be to created the nested documents (see `common.MapStr.Put(...)`). This brings the risk in case a value and a nested object collide. Example `a.b = c; a.b.d = f` This is not possible.

Unrelated to the above, as this is a dynamic metricset, there should be an example json file.